pkg/runtime/validate_test.go

```diff
@@ -72,6 +72,22 @@ func TestValidateInputs(t *testing.T) {
 			},
 			wantErr: errDirNotExists,
 		},
+		{
+			// should error out in validations if -f option is not a file
+			name: "valid directory passed as file path",
+			executor: Executor{
+				filePath: "./testdata/testdir",
+			},
+			wantErr: errNotValidFile,
```

Comment:
You changed the type of `wantErr` to boolean everywhere else. and here you're keeping the type as `error` . That's inconsistent.

Reply:
I modified the failing tests @dev-gaur.